catalog/task/collect-data/0.3/collect-data.yaml

```diff
@@ -63,15 +75,20 @@ spec:
 
         cd $SUBDIR
 
+        echo -n "$(params.subdirectory)/release.json" > $(results.release.path)
```

Comment:
I wonder if we could streamline this a bit. How about we ditch the `SUBDIR` env var and do this:
```sh
if [ -n "$(params.subdirectory)" ]; then
  mkdir -p $(workspaces.data.path)/$(params.subdirectory)
fi

RELEASE_PATH=$(params.subdirectory)/release.json
echo -n $RELEASE_PATH > $(results.release.path)
get-resource "release" "$RELEASE" > $(workspaces.data.path)/$RELEASE_PATH
```

And similar for the rest.

But only if you find it better like I do :)

Reply:
I don't have much of an opinion on it, so I changed it to like you said

Reply:
ok, cool :)